components/core/src/clp/streaming_archive/ArchiveMetadata.hpp

```diff
@@ -102,6 +118,12 @@ class ArchiveMetadata {
     // The size of the archive
     uint64_t m_compressed_size{0};
     uint64_t m_dynamic_compressed_size{0};
+    // TODO: The following fields are used in single-file archive; however, they are not
+    // currently part of multi-file archive metadata. Modifying multi-file archive metadata
+    // disk format is potentially a breaking change and not currently required.
+    std::string m_variable_encoding_methods_version{ffi::cVariableEncodingMethodsVersion};
```

Comment:
I guess a high level question is. how are those constants related to multi-file-archive?

Technically, since they are not used by anything else, they can also be directly defined under struct "single_archive_metadata"?